toolbox/backfill_content_type/.gitignore

```diff
@@ -0,0 +1,4 @@
+extracts/*
+!extracts/*.md
```

Comment:
TIL there's a negation operator you can use here - this is a good pattern for "nothing in here except for this particular thing."